sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -0,0 +1,171 @@
+# Azure Form Recognizer client library for Python
+
+* TODO
+* TODO
+
+[Source code][TODO] | [Package (PyPI)][TODO] | [API reference documentation][TODO]| [Product documentation][TODO] | [Samples][TODO]
+
+## Getting started
+
+### Prerequisites
+* Python 2.7, or 3.5 or later is required to use this package.
+* You must have an [Azure subscription][azure_subscription] and a
+[Cognitive Services or Form Recognizer resource][FR_or_CS_resource] to use this package.
+
+### Install the package
+Install the Azure Form Recognizer client library for Python with [pip][pip]:
+
+```bash
+pip install azure-ai-formrecognizer --pre
+```
+
+### Create a Form Recognizer resource
+Form Recognizer supports both [multi-service and single-service access][multi_and_single_service].
+Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Form Recognizer access only, create a Form Recognizer resource.
+
+You can create the resource using
+
+**Option 1:** [Azure Portal][TODO]
+
+**Option 2:** [Azure CLI][TODO].
+Below is an example of how you can create a Form Recognizer resource using the CLI:
+
+```bash
+# Create a new resource group to hold the form recognizer resource -
+# if using an existing resource group, skip this step
+az group create --name my-resource-group --location westus2
+```
+
+```bash
+# Create form recognizer
+az cognitiveservices account create \
+    --name form-recognizer-resource \
+    --resource-group my-resource-group \
+    --kind FormRecognizer \
+    --sku F0 \
+    --location westus2 \
+    --yes
+```
+
+### Authenticate the client
+
+
+#### Looking up the endpoint
+You can find the endpoint for your form recognizer resource using the
+[Azure Portal][azure_portal_get_endpoint]
+or [Azure CLI][azure_cli_endpoint_lookup]:
+
+```bash
+# Get the endpoint for the form recognizer resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
+```
+
+#### Types of credentials
+The `credential` parameter may be provided as a `FormRecognizerApiKeyCredential`.
+See the full details regarding [authentication][cognitive_authentication] of cognitive services.
+
+To use an [API key][cognitive_authentication_api_key],
+pass the key as a string into an instance of `FormRecognizerApiKeyCredential("<api_key>")`.
+The API key can be found in the Azure Portal or by running the following Azure CLI command:
+
+```az cognitiveservices account keys list --name "resource-name" --resource-group "resource-group-name"```
+
+Use the key as the credential parameter to authenticate the client:
+```python
+from azure.ai.formrecognizer import FormRecognizerClient, FormRecognizerApiKeyCredential
```

Comment:
Relation with https://github.com/Azure/azure-sdk-for-python/pull/10509 ?

Reply:
Opened an issue to replace my service-specific credential type/policy with the azure-core one once that PR merges: https://github.com/Azure/azure-sdk-for-python/issues/10540